content/docs/user-guide/best-practices.md

```diff
@@ -0,0 +1,135 @@
+# Best Practices for DVC Projects
+
+DVC provides a systematic approach towards managing and collaborating on data
+science projects. You can manage your projects with DVC more efficiently using
+the practices listed here:
+
+## Source code and data versioning
+
+You can use DVC to avoid discrepancies between
+[revisions](https://git-scm.com/docs/revisions) of source code and
+[versions](/doc/use-cases/versioning-data-and-model-files) of data files. DVC
+replaces all large data files, models, etc. with small
+[metafiles](doc/user-guide/dvc-files-and-directories) (tracked by Git). These
+files point to the original data, which you can access by first checking out the
+required `revision` using Git followed by `dvc checkout` to update DVC tracked
+data files/dir:
+
+```dvc
+$ git checkout 95485f   # Git commit of required data version
+$ dvc checkout
+```
+
+If your dataset consist of multiple files like images, etc. then the best way to
+track whole directory is with single `.dvc` file. You can use `dvc add` with
+relative path to directory:
+
+```dvc
+$ dvc add data/images
+```
+
+## Experiments and tracking parameters
+
+You can use DVC for tuning [parameters](doc/command-reference/params), improving
+target [metrics](doc/command-reference/metrics) and visualizing the changes with
+[plots](doc/command-reference/plots). In the first step tune parameters in
+default `params.yaml` file and reproduce the pipeline:
```

Comment:
Not bad! But per https://github.com/iterative/dvc.org/pull/1705#pullrequestreview-477137236 a previous best practice about how to organize DVC project experiments with Git is needed. The 2 basic options are: as commits in a single branch (plus tags), and as multiple branches (one per experiment).